docker-compose.yaml

```diff
@@ -23,9 +20,32 @@ services:
     ports:
       - 9000:9000
 
+  pmm-client:
+    container_name: pmm-client
+    image: centos
+    depends_on:
+      - pmm-server
+      - sysbench-ps
+    command: >
+      bash -c "
+        yum install https://repo.percona.com/yum/percona-release-latest.noarch.rpm -y
```

Comment:
maybe it's better to make Dockerfile to not run this script each time.

Reply:
I think it is better for all of us (devs) to learn how to setup pmm-admin / pmm-agent locally. @BupycHuk since you are near @askomorokhov now, can you show him? :)

Reply:
This looks like holly war :). I know how to step them on linux - pmm-stages. But I do not see quite strong reason to setup it on my Mac -  such env is far from usual prod env. :)

Reply:
I will be _way_ faster, especially on your MacBook. It is also much more easier to develop and debug it.
But I do not insist.